README.md

```diff
@@ -215,7 +215,9 @@ In the example above, the `Target` attribute is specified three different ways,
 
 #### Modeling Security Providers
-One place where the semantics are different is for WebLogic security providers.  Because provider ordering is important, and to make sure that the ordering is correctly set in the newly created domain, the Create Domain Tool will look for security providers of each base type (for example, Authentication Providers, Credential Mappers, and such) to see if any are included in the model.  If so, the tool will make sure that  the providers only listed for a type are present in the resulting domain so that the providers are created in the necessary order.  For example, if the model specified an `LDAPAuthenticator` and an `LDAPX509IdentityAsserter` similar to what is shown below, the `DefaultAuthenticator` and `DefaultIdentityAsserters` will be deleted.  If no providers for a base type are listed in the model, then the default providers will be left untouched.
+WebLogic Server Security Configuration requires special handling and causes the need for the model semantics to differ from other folders.  Because provider ordering is important, and to make sure that the ordering is correctly set in the newly created domain, the Create Domain Tool and Update Domain Tool require all providers be specified in the model for any provider type that will be created or altered.  For example, if you want to change one of the providers in the provider type AuthenticationProvider, your model must specify all of the AuthenticationProvider providers and any non-default attributes for those providers.  In order to apply security providers, these tools will delete all providers from the target domain for those provider types specificed in the model before adding the providers from the model to the target domain. Provider types that are omitted from the model will be unchanged.  Example provider types are Adjudicator, AuthenticationProvider, Authorizer, CertPathProvider, CredentialMapper, PasswordValidator, and RoleMapper.
```

Comment:
WebLogic Server Security Configuration -> WebLogic Server security configuration
require all providers be specified -> require that all providers be specified 
AuthenticationProvider -> `AuthenticationProvider` (globally for all provider types), for example, `Adjudicator`, `AuthenticationProvider`, `Authorizer`, `CertPathProvider`, `CredentialMapper`, `PasswordValidator`, and `RoleMapper`

Reply:
updated in latest commit, thanks